robowflex_library/yaml/test_cob4.yml

```diff
@@ -0,0 +1,41 @@
+name: (noname)
```

Comment:
This entry (the robot state) throws warnings errors in Rviz. You can either fix the errors or you can delete the robot_state in the scene completely it is not needed or used.

Reply:
done!